main/init/package.mill

```diff
@@ -86,4 +93,53 @@ object `package` extends RootModule with build.MillPublishScalaModule {
     )
     def testModuleDeps = super.testModuleDeps ++ Seq(build.scalalib, buildgen.test)
   }
+
+  object sbt extends build.MillPublishScalaModule {
+    def moduleDeps = Seq(buildgen, models(build.Deps.scalaVersion))
+
+    val sbtPluginProjectPath = millModuleBasePath.value / "sbt-mill-init-export-build"
```

Comment:
Could we build this plugin using Mill instead of SBT? A SBT plugin is just a `ScalaModule` with some dependencies configured after all, so we should be able to integrate it into Mill as a normal `ScalaModule` without needing to shell out to `sbt`

Reply:
I first tried this but it didn't work well. You can see from the commit message of 3c9f155d1fd5e4146e22284d7104e02da83bbbcf that after reviewing the contents of https://repo1.maven.org/maven2/com/eed3si9n/sbt-assembly_2.12_1.0/2.3.1/sbt-assembly_2.12_1.0-2.3.1.jar, I see there is an "sbt/sbt.autoplugins" file which has the contents:

```
sbtassembly.AssemblyPlugin

```

This probably means that such a file has to be included for an sbt plugin and a plain Mill module doesn't support this. I guess if using Mill a task could be added to write the file to the jar, but I was afraid that there were other files packaged in the jar that I am not aware of, so I switched to sbt.

Reply:
Let's try to build it with Mill. Adding resources that will be included in the jar is easy enough (https://mill-build.org/mill/scalalib/module-config.html#_common_configuration_overrides), so we just need to look at what the SBT-build jar contains and make sure our Mill-built jar has the same contents

Reply:
OK I will give this a try.